script/room.js

```diff
@@ -331,9 +331,43 @@ var Room = {
 					'sulphur': 50
 				};
 			}
+		},
+		'i dagger': {
+			name: _('dagger'),
+			type: 'weapon',
+			buildMsg: _("good for melee"),
+			cost: function() {
```

Comment:
These daggers seem a little cheap for being 1 damage below the swords and the fact that they add another weapon to stack damage on

Reply:
Yeah, I guess so. You can increase the costs of them.